### PR TITLE
Increment document information

### PR DESCRIPTION
**Pull request by Igoraamc**
What does this implement/fix? Explain your changes.
---------------------------------------------------
This pull request increments the README.md with the new Go API endpoints.

- Add all the API information that the user might need to see 🤓 

Does this close any currently open issues?
------------------------------------------
Any relevant logs, error output, etc?
-------------------------------------
Any other comments?
-------------------
Where has this been tested?
---------------------------

**Operating System/Platform:** MACOS Catalina

**GO Version:** go 1.13.7 darwin/amd64

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,3 +102,244 @@ This will start the server with auto-reload.
...
+
+Field | Required | Description
+--- | --- | ---
+auth_token | Y | User's authentication token
+
+### Body Parameters
+
+Field | Required | Description
+--- | --- | ---
+data | Y | The message fata sent by the device. This data includes the device that it was sent and the payload.
```
Comment (netoax) [inline README.md]:
```suggestion
data | Y | The message data sent by the device. It includes both sender identification and payload.
```

File: README.md [hunk 2]
```diff
@@ -102,3 +102,244 @@ This will start the server with auto-reload.
...
+POST https://storage.knot.cloud/data
+```
+
+##### Request Body
+```json
+{
+	"from": "188824f0-28c4-475b-ab36-2505402bebcb",
+	"payload": {
+	    "sensor_id": 2,
+    	"value": 234
```
Comment (netoax) [inline README.md]: Please, fix indentation.

File: README.md [hunk 3]
```diff
@@ -102,3 +102,244 @@ This will start the server with auto-reload.
...
 curl http://<hostname>:<port>/healthcheck
 ```
+
+## API
+
+```bash
+POST /data
+```
+
+Stores the device message received from the webhook service.
```
Comment (netoax) [inline README.md]:
```suggestion
Stores the device messages. 
```

File: README.md [hunk 4]
```diff
@@ -102,3 +102,244 @@ This will start the server with auto-reload.
...
+	"from": "188824f0-28c4-475b-ab36-2505402bebcb",
+	"payload": {
+	    "sensor_id": 2,
+    	"value": 234
+	}
+}
+```
+
+##### Response
+```json
```
Comment (netoax) [inline README.md]: We don't return this response anymore, just consider `201 - created` as [here](https://github.com/CESARBR/knot-cloud-storage#response).

File: README.md [hunk 5]
```diff
@@ -102,3 +102,244 @@ This will start the server with auto-reload.
...
+    }
+]
+```
+
+---
+
+```
+GET /data/{deviceID}
+```
+Get the messages received from the webhook service by a specific device.
```
Comment (netoax) [inline README.md]:
```suggestion
Get the messages by a specific device.
```

File: README.md [hunk 6]
```diff
@@ -102,3 +102,244 @@ This will start the server with auto-reload.
...
+    }
+]
+```
+
+---
+
+```
+GET /data/{deviceID}/sensor/{sensorID}
+```
+Get the messages received from the webhook service by a specific device sensor.
```
Comment (netoax) [inline README.md]:
```suggestion
Get the messages by a specific device sensor.
```

File: README.md [hunk 7]
```diff
@@ -102,3 +102,244 @@ This will start the server with auto-reload.
...
+---
+
+```
+DELETE /data
+```
+Delete all the device messages.
+
+### Parameters
+#### Header Parameters
+In order to get the device messages, you need to be authenticated as its owner.
```
Comment (netoax) [inline README.md]:
```suggestion
In order to delete the messages, you need to be authenticated as a valid user. Only data from the things which are owned by this user will be removed.
```

File: README.md [hunk 8]
```diff
@@ -102,3 +102,244 @@ This will start the server with auto-reload.
...
+---
+
+```
+DELETE /data/{deviceID}
+```
+Delete all messages from a specific device.
+
+### Parameters
+#### Header Parameters
+In order to get the device messages, you need to be authenticated as its owner.
```
Comment (netoax) [inline README.md]:
```suggestion
In order to delete the device messages, you need to be authenticated as its owner.
```

**Review by netoax — CHANGES_REQUESTED** [review]
Good patch, just some comments to help you improve it.

File: README.md (hunk 1 above)
Comment (Igoraamc) [reply README.md]: Done!

File: README.md (hunk 3 above)
Comment (Igoraamc) [reply README.md]: Done!

File: README.md (hunk 4 above)
Comment (Igoraamc) [reply README.md]: Done!

File: README.md (hunk 5 above)
Comment (Igoraamc) [reply README.md]: Done!

File: README.md (hunk 6 above)
Comment (Igoraamc) [reply README.md]: Done!

File: README.md (hunk 7 above)
Comment (Igoraamc) [reply README.md]: Done!

File: README.md (hunk 8 above)
Comment (Igoraamc) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (Igoraamc) [reply README.md]: Done!

**Review by netoax — APPROVED** [review]
LGTM!

**Pull request merged**